src/halmos/processes.py

```diff
@@ -193,6 +197,24 @@ def submit(self, future: PopenFuture) -> PopenFuture:
             future.start()
             return future
 
+    def interrupt(self, tag: str) -> None:
+        """Interrupts all futures with the specified tag.
+        
+        Args:
+            tag: The tag identifying futures to interrupt. 
+                 Futures without tags are not affected.
```

Comment:
We don't want Futures without tags, change this to `Futures with a different tag are not affected.`